src/core/Microsoft.PowerToys.Settings.UI/Views/ImageResizerPage.xaml

```diff
@@ -99,21 +127,19 @@
                 </ListView.ItemContainerStyle>
                 <ListView.ItemTemplate>
                     <DataTemplate x:Name="SingleLineDataTemplate" x:DataType="models:ImageSize" >
-                        <StackPanel Name="ImageResizer_Configurations" x:Uid="ImageResizer_Configurations" Orientation="Horizontal" Height="48" Padding="0" Spacing="4">
+                        <StackPanel Name="ImageResizer_Configurations" x:Uid="ImageResizer_Configurations" Orientation="Horizontal" Padding="0" Spacing="4">
                             <TextBox Name="ImageResizer_Name"
                                      x:Uid="ImageResizer_Name"
                                      Text="{x:Bind Path=Name, Mode=TwoWay}" 
                                      Width="90"
                                      VerticalAlignment="Center"
-                                     Height="34"
                                      Margin="{StaticResource SmallTopMargin}"/>
 
                             <ComboBox Name="ImageResizer_Fit_Property"
                                       x:Uid="ImageResizer_Fit_Property"
                                       SelectedIndex="{x:Bind Path=Fit, Mode=TwoWay}"
                                       Width="90"
                                       VerticalAlignment="Center"
-                                      Height="34"
```

Comment:
I'd go for MinHeight instead of removing it entirely to make sure it renders properly when setting display scaling to e.g. 100%/150%

Reply:
This isn't the issue, they scale fine by default, the issue is the snap zone.
![image](https://user-images.githubusercontent.com/1462282/96911547-d5ff8300-1455-11eb-832f-dfe7f51cdb99.png)
this is at 200dpi